velox/substrait/README.md

```diff
@@ -0,0 +1,8 @@
+# Velox substrait upgrade
+
+Velox copy substrait proto files from `https://github.com/substrait-io/substrait/tree/main/proto/substrait` to generate substrait cpp files. So if you want to upgrade substrait to a certain version, follow the steps like below:
```

Comment:
- "Velox includes a copy of the Substrait proto files"

- Remove "like" from "follow the steps like below" -> "follow the steps below"

Reply:
Done